spec/assessments/calculateFleschReadingSpec.js

```diff
@@ -0,0 +1,17 @@
+var fleschReadingAssessment = require( "../../js/assessments/calculateFleschReading.js" );
+var Paper = require( "../../js/values/Paper.js" );
+
+var factory = require( "../helpers/factory.js" );
+var i18n = factory.buildJed();
+
+describe( "An assessment for the fleschReading", function(){
```

Comment:
Perhaps add more tests to also touch the other possible scoring outcomes.
Testing of other messages that are quite similar to this one, isn't necessary.